inbm/dispatcher-agent/dispatcher/sota/os_factory.py

```diff
@@ -99,13 +99,15 @@ def create_os_updater(self) -> OsUpdater:
         pass
 
     @abstractmethod
-    def create_snapshotter(self, sota_cmd: str, snap_num: Optional[str], proceed_without_rollback: bool) -> Snapshot:
+    def create_snapshotter(self, sota_cmd: str, snap_num: Optional[str],
```

Comment:
Ensure that the `reboot_device` parameter is documented in the docstrings of the `create_snapshotter` method in each OS-specific factory class. [medium]

Reply:
It's documented at the parent level, but added it anyways for that one.